### DISCUSSION TITLE
NEW Russian translations + Morecolor

### DISCUSSION THREAD

**Discussion by denzel519** [J-Tanzanite/Little-Anti-Cheat#176] [General]
In lilac.sp
add after
#include <convar_class>
This
#include <morecolors>

In lilac_stock.sp
replace
for (int i = 0; i < n; i++)
PrintToChat(admins[i],

On
for (int i = 0; i < n; i++)
CPrintToChat(admins[i],

And you will be able to color the text in the translation.

New translation file for RU
No colors - https://cloud.mail.ru/public/x94J/FkKG7z2VJ
Colors - https://cloud.mail.ru/public/MnHj/1nG1Com5J

**Comment by J-Tanzanite** [comment]
I'm a little unsure about using colors in chat messages.
Not because I'm against it, I just haven't looked into how they work and what compiling would look like, I'm assuming it's just a header and wouldn't cause much problems in setup.

But having colors built into the default translations, I'm not too sure about that; that ought to be a server by server choice (plus their own theming).
It would be much preferable if you posted the translations on github or pastebin (recommended).

Sorry for the slow reply btw, been busy with IRL stuff.
